surveys/models.py

```diff
@@ -1,3 +1,36 @@
 from django.db import models
+from django.db.models import Sum
 
-# Create your models here.
+"""
+    Shamefully copying and pasting from the django intro tutorial 🤣
+    and pridefully extending from there
+"""
+
+
+class Question(models.Model):
+    question_text = models.CharField(max_length=200)
+    pub_date = models.DateTimeField('date published')
+
+    def __str__(self):
+        return self.question_text
+
+    def choices_count(self):
```

Comment:
Considering that Django ORM is lazy I'm kinda worried about the performance of this when we have more data 😟

Reply:
I know, but let's worry about that later **Premature Optimization Is the Root of All Evil** let's create an issue for later refactoring

Reply:
Created #2  to remember let's merge for now